### PR TITLE
Magiclysm proficiencies

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary

SUMMARY: Mods "Adds proficiencies to magiclysm recipes"

#### Purpose of change

Magiclysm has a lot of unique crafting that would benefit frmo proficiency support.

#### Describe the solution

Requires #44446

Adds existing proficiencies to magiclysm recipes

Adds alchemy and almetallurgy proficiencies for crafting potions and for making items with orichalcum.

Adds bowyer and fletcher proficiencies to mainline, currently not used except for magiclysm but I'll get right on that in my copious free time.

#### Testing
image

image

#### Additional context
AFAICT we can't make proficiencies granted by mutations yet, which is too bad. That should be fixed, because attunement traits would go great with several proficiencies.

### CODE REVIEW

**Comment by I-am-Erk** [comment]
Leaving in draft until requisite PRs are in.

**Comment by I-am-Erk** [comment]
@KorGgenT Any complaints before I merge?

**Comment by KorGgenT** [comment]
was waiting for it to undraft. will review it later tonight or tomorrow

File: data/mods/Magiclysm/recipes/blacksmithing.json [hunk 1]
```diff
@@ -49,9 +64,10 @@
     "subcategory": "CSC_OTHER_OTHER",
     "skill_used": "spellcraft",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/recipes/blacksmithing.json:65]:
```suggestion
    "skill_used": "chemistry",
```
chemistry with alchemy makes way mor sense now that chemistry is a thing.

**Review by KorGgenT — APPROVED** [review]
Looks good, had to change one thing while i looked through it so the tests had to rerun. I'll check back later to merge it.

**Comment by faefux** [comment]
Hi, @I-am-Erk, two questions. Are you planning to add prof_bowyery to bow recipes any time soon? And if not, are crossbow recipes intended to fall under prof_bowyery as well as regular bows or will we be using a new proficiency for that?

**Comment by I-am-Erk** [comment]
Now that this is merged, adding prof_bowyery is going to be done as soon as my file cleanup pr is in

**Pull request merged**